tfe/resource_tfe_organization_token_test.go

```diff
@@ -215,3 +276,36 @@ resource "tfe_organization_token" "regenerated" {
   force_regenerate = true
 }`, rInt)
 }
+
+func testAccTFEOrganizationToken_existsWithoutExpiry(rInt int) string {
+	return fmt.Sprintf(`
+resource "tfe_organization" "foobar" {
+  name  = "tst-terraform-%d"
+  email = "admin@company.com"
+}
+
+resource "tfe_organization_token" "foobar" {
+  organization = tfe_organization.foobar.id
+}
+
+resource "tfe_organization_token" "error" {
```

Comment:
❓ why would this configuration result in an error? It looks like we have set [ExpiredAt](https://github.com/hashicorp/terraform-provider-tfe/pull/844/files#diff-3ba6ff1c80cd647a2c785e0c5f5fb72a8aa985181fb2689b41686131d98bfb0fR45) as an optional attibute (which I think is correct!). So I would expect that a configuration like this would create a token which just doesnt expire